### DISCUSSION TITLE
Image preview broken

### DISCUSSION THREAD

**Discussion by Lemonbonbon** [James-Yu/LaTeX-Workshop#3145] [Q&A]
The image preview is broken suddenly. Normally if hover with the mouse over "includegraphics{...}" a preview of the image is shown. Now, there is only displayed broken text, which looks like the image is displayed in hex or something. What could cause this? Is this related to this extension? How do I solve this?

**Comment by tamuratak** [comment]
- Fill out all the information required in [the issue template](https://github.com/James-Yu/LaTeX-Workshop/blob/master/.github/ISSUE_TEMPLATE/bug-report.md). 
- Please paste the whole log messages here, not parts of ones. It should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

![スクリーンショット 2020-02-14 20 41 25](https://user-images.githubusercontent.com/10665499/74528024-87b2ef80-4f6a-11ea-9c5d-4f9feff25524.png)

When all of the above is done, we will reopen this issue if needed.

**Comment by tamuratak** [comment]
Please fill out the [issue template](https://github.com/James-Yu/LaTeX-Workshop/blob/master/.github/ISSUE_TEMPLATE/bug-report.md) next time.
